etc/config/wasm.defaults.properties

```diff
@@ -0,0 +1,21 @@
+compilers=&wasmtime
+objdumper=llvm-objdump
+objdumperType=llvm
```

Comment:
I used `llvm-objdump` here because wasmtime supports a `--target=<triple>` parameter that cross-compiles WASM modules for other architectures. The problem is that when using `objdump` it then can't disassemble the resulting file. I'm not sure if there is a better solution to handle these cases